charts/kkr/wallet-api/templates/deployment.yaml

```
@@ -0,0 +1,181 @@
+            - name: AWS_WALLET_CREATE_NOTIFICATION_QUEUE_URL
+              valueFrom:
+                configMapKeyRef:
+                  key: aws-wallet-create-notification-queue-url
+                  name: {{ include "wallet-api.fullname" . }}-aws
+            - name: OCTET_API_TOKEN
+              valueFrom:
+                secreteKeyRef:
+                  key: octet-api-token
+                  name: {{ include "wallet-api.fullname" . }}-octet
+            - name: OCTET_BASE_URL
+              valueFrom:
+                secretKeyRef:
+                  key: octet-base-url
+                  name: {{ include "wallet-api.fullname" . }}-octet
+            - name: WALLET_IDX
+              valueFrom:
+                secretKeyRef:
+                  key: wallet-idx
+                  name: {{ include "wallet-api.fullname" . }}-octet
+            - name: RECOVERY_PIN
+              valueFrom:
+                secretKeyRef:
+                  key: recovery-pin
+                  name: {{ include "wallet-api.fullname" . }}-octet
+            - name: ALCHEMY_API_KEY
+              valueFrom:
+                secretKeyRef:
+                  key: alchemy-api-key
+                  name: {{ include "wallet-api.fullname" . }}-alchemy
+            - name: ALCHEMY_HOST
+              valueFrom:
+                secretKeyRef:
+                  key: alchemy-host
+                  name: {{ include "wallet-api.fullname" . }}-alchemy
+            - name: JWT_EXPIRES_IN
+              valueFrom:
+                configMapKeyRef:
+                  key: jwt-expires-in
+                  name: {{ include "wallet-api.fullname" . }}-jwt
+            - name: JWT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  key: jwt-secret
+                  name: {{ include "wallet-api.fullname" . }}-jwt
+            - name: JWT_AUTH_SECRET
+              valueFrom:
+                secretKeyRef:
+                  key: jwt-auth-secret
+                  name: {{ include "wallet-api.fullname" . }}-jwt
+            - name: HASH_IDS_KEY
+              valueFrom:
+                secretKeyRef:
+                  key: hash-ids-key
+                  name: {{ include "wallet-api.fullname" . }}-encoding
+            - name: RSA_PRIVATE_KEY
+              valueFrom:
+                secretKeyRef:
+                  key: rsa-private-key
+                  name: {{ include "wallet-api.fullname" . }}-encoding
+            - name: RSA_PUBLIC_KEY
+              valueFrom:
+                secretKeyRef:
+                  key: rsa-public-key
+                  name: {{ include "wallet-api.fullname" . }}-encoding
+            - name: CHAIN_ID
+              valueFrom:
+                secretKeyRef:
+                  key: chain-id
+                  name: {{ include "wallet-api.fullname" . }}-common
+          ports:
+            - name: http
+              containerPort: 80
+              protocol: TCP
```

`kkr-wallet-api`는 (아마도) PORT 3000 으로 서비스되니까, 이 값을 `3000` 으로 변경해야합니다!
```suggestion
            - name: http
              containerPort: 3000
              protocol: TCP
```